### PR TITLE
[tests-only] Change smoke test to e2e test in .drone.star

### PR DESCRIPTION
**Pull request by phil-davis**
## Description
The playwright tests folder was recently renamed from `smoke` to `e2e`. Do similar in drone CI to avoid future confusion.

## How Has This Been Tested?
CI

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests

## Checklist:
- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link>

### CODE REVIEW

File: .drone.star [hunk 1]
```diff
@@ -2966,16 +2966,16 @@ def skipIfUnchanged(ctx, type):
             "^__mocks__/.*",
             "^packages/.*/tests/.*",
             "^tests/integration/.*",
-            "^tests/smoke/.*",
+            "^tests/e2e/.*",
```
Comment (phil-davis) [inline .drone.star:2969]: Note: this is the only real change. It should make the detection of changes to combinations of e2e and acceptance tests correct again.

The other changes are just names, they do not effect the actual functionality of the drone starlark script.

**Review by pascalwengerter — APPROVED** [review]
Great catch, thanks!

**Comment by ownclouders** [comment]
Results for <strong>oCISSharingInternal3</strong> https://drone.owncloud.com/owncloud/web/21529/59/1
The following scenarios passed on retry:
- webUISharingInternalGroupsSharingIndicator/shareWithGroups.feature:136

**Comment by ownclouders** [comment]
Results for <strong>oC10SharingPublic1</strong> https://drone.owncloud.com/owncloud/web/21529/37/1
The following scenarios passed on retry:
- webUISharingPublicExpire/shareByPublicLinkExpiringLinks.feature:11

**Review by individual-it — APPROVED** [review]

**Pull request merged**